libcaf_core/caf/node_id.test.cpp

```diff
@@ -49,8 +49,6 @@ T unbox(std::optional<T> x) {
   return std::move(*x);
 }
 
-} // namespace
-
 #define CHECK_PARSE_OK(str, ...)                                               \
```

Comment:
Defines are an old C construct. We never put macros into namespaces by convention, because macros are not affected by namespaces in the first place.